mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -645,3 +648,79 @@ def __call__(self, results):
         results['target_weight'] = target_weight
 
         return results
+
+
+@PIPELINES.register_module()
+class TopDownGenerate3DHeatmapTarget:
```

Comment:
How about moving this to pose3d_transform.py?

Reply:
Fixed